ios/brave-ios/Sources/Brave/Frontend/Browser/TabManager.swift

```diff
@@ -83,6 +82,11 @@ class TabManager: NSObject {
     return TabManager.getNewConfiguration()
   }()
 
+  private var policyDeciders: [any TabWebPolicyDecider] = []
+  func addPolicyDecider(_ policyDecider: any TabWebPolicyDecider) {
+    policyDeciders.append(policyDecider)
+  }
+
```

Comment:
Missed removing these, should be removed and re-added in the future when we support multiple policy deciders